src/App.js

```diff
@@ -5,6 +5,7 @@ import Login from './Login/Login';
 import ErrorPage from './ErrorPage/ErrorPage';
 
 import React, { Component } from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
```

Comment:
The correct use of BrowserRouter is wrapping your entire application in the BrowserRouter tags - which _is_ what you are doing! However, this should be implemented in the index.js

Reply:
I see, that makes perfect sense. I will move that over to the index.js. Thanks Casey!